src/TPPCore.Service.Common/TPPCore.Service.Common.csproj

```diff
@@ -8,6 +8,7 @@
     <PackageReference Include="CommandLineParser" Version="2.1.1-beta" />
     <PackageReference Include="log4net" Version="2.0.8" />
     <PackageReference Include="Microsoft.AspNetCore" Version="2.0.1" />
+    <PackageReference Include="RingBuffer" Version="1.5.1" />

```

Comment:
Any reason why RingBuffer is included in Common? I can only remember using this in Irc.Tests.